swagger.json

```diff
@@ -537,18 +614,61 @@
               }
             }
           },
-          "201": {
-            "description": "Sign-off record created or updated",
+          "409": {
+            "description": "Submitted parameters do not match parameters for existing sign-off associated with the same case",
```

Comment:
Parameters may or may not match an existing sign-off, and either way should be ok. What would cause this status? Same question for the other sign-off endpoints too

Reply:
Missed one instance of this